contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -67,6 +67,7 @@ Examples of features which are not currently eligible for conformance tests:
 - cloud-provider-specific features, eg: GCE monitoring, S3 Bucketing, etc.
 - anything that requires a non-default admission plugin
 - features that are pending deprecation, eg: componentstatus
+- any endpoints that are used as tools for operation like apiserver logs should not be part of conformance. 
```

Comment:
Can we be a bit more descriptive:

 - any endpoints that are operational tools, rather than for application-oriented, should not be part of conformance, eg: apiserver logs

Reply:
Updated. Thank you @johnbelamaric